integration/plugin/logging/cmd/discard/main_test.go

```diff
@@ -0,0 +1 @@
+package main
```

Comment:
unneeded file?

Reply:
I think go will complain if there is no test file in the directory.